internal/server/singleprocess/state/status_report.go

```diff
@@ -0,0 +1,61 @@
+package state
+
+import (
+	pb "github.com/hashicorp/waypoint/internal/server/gen"
+)
+
+var statusReportOp = &appOperation{
+	Struct: (*pb.StatusReport)(nil),
+	Bucket: []byte("statusreport"),
+
+	MaximumIndexedRecords: 1, // Only store the "latest" status report
```

Comment:
Should we also be pruning the records out of boltdb then too?

Reply:
RE: video review, this is doing the right thing by storing 1 report.